tests/testMakePixelToTanPixel.py

```diff
@@ -26,14 +26,73 @@
 """
 import unittest
 
-import numpy
+import math
 
 import lsst.utils.tests
 import lsst.afw.geom as afwGeom
 import lsst.afw.cameraGeom as cameraGeom
 from lsst.afw.cameraGeom import makePixelToTanPixel
 
-class MakePixelToTanPixelTestCaseCase(unittest.TestCase):
+class MakePixelToTanPixelTestCaseCase(lsst.utils.tests.TestCase):
+    def testSimpleCurvedFocalPlane(self):
```

Comment:
How useful is it to have a test of this simple curved focal plane?  Isn't the same functionality being tested in the curved focal plane test below.  I think it was helpful for me trying to figure what is going on, but I'm not sure it adds anything new to the test suite.